src/widgets/list.rs

```diff
@@ -1301,7 +1472,7 @@ mod tests {
             &single_item,
             Some(1),
             [
-                "  Item 0  ",
+                ">>Item 0  ",
```

Comment:
This is a change that's likely more correct than the existing approach, but is probably a breaking change and should be noted as such.

Reply:
Agreed, in my lists I do the same by default. This feels much more natural. If consumers don't implement this manually - for me personally this always leads to confusing interactions.